packages/flutter_tools/lib/src/runner/flutter_command_runner.dart

```diff
@@ -390,4 +390,33 @@ class FlutterCommandRunner extends CommandRunner {
     if (ArtifactStore.flutterRoot == null)
       ArtifactStore.flutterRoot = _defaultFlutterRoot;
   }
+
+  /// Get all pub packages in the Flutter repo.
+  List<Directory> getRepoPackages() {
+    String root = path.absolute(ArtifactStore.flutterRoot);
+    List<String> directories = <String>[];
+
+    // dev/ (excluding remove dev/docs)
+    directories.addAll(_gatherProjectPaths(path.join(root, 'dev')));
+    directories.removeWhere((String dir) => dir.endsWith('docs'));
```

Comment:
A synthetic one is created as part of doc generation (`dev/dartdoc.dart`), but this directory isn't meant to be treated as a pub package.